docs/_themes/clusterhq/layout_docs.html

```diff
@@ -0,0 +1,197 @@
+            })();
+        </script>
+    {% endif %}
+</head>
+<body>
+    
+    <!-- Menu -->
+    <div class="menubar">
+        <div class="container">
+            <div class="row">
+                <div class="col-sm-12 col-md-3 col-xs-6 logo">
+                    <a href="https://clusterhq.com/">
+                        <img src="{{ pathto('_static/images/logo@2x.png', 1) }}" width="200" height="50">
+                    </a>
+                </div>
+                <div class="visible-xs-block col-xs-6 mob-nav">
+                    <a href="javascript:$('nav ul').toggleClass('hidden-xs');">
+                        <img src="{{ pathto('_static/images/menu@2x.png', 1) }}" width="60" height="47" class="pull-right"/>
+                    </a>
+                </div>
+                <nav class="clearfix">
+                    <div class="col-sm-12 col-md-9">
+                        <ul class="menu hidden-xs">
+                            <li><a href="https://docs.clusterhq.com/en/latest/gettingstarted/index.html">Getting Started</a></li>
+                            <li class="current-menu-item"><a href="https://docs.clusterhq.com/en/latest/">Docs</a></li>
+                            <li><a href="https://clusterhq.com/solutions/">Solutions</a></li>
+                            <li><a href="https://clusterhq.com/about/">About</a></li>
+                            <li><a href="https://clusterhq.com/careers/">Careers</a></li>
+                            <li class="button menu hidden-xs"><a href="https://docs.clusterhq.com/en/latest/gettingstarted/index.html">Try Flocker</a></li>
+                        </ul>
+                    </div>
+                </nav>
+            </div>
+        </div>
+    </div>
+    
+    <!-- Contents -->
+    <div class="container contents">
+        <div class="row">
+            <!-- Contents menu -->
+            <div class="col-sm-3 page contents-menu">
+                <div class="visible-xs-block mob-contents">
+                    Contents
+                    <div class="pull-right plus">
+                        <a href="javascript:$('.contents-menu>ul').slideToggle();">+</a>
+                    </div>
+                </div>
+                <!-- Menu area start -->
+                {{ toctree(collapse=false, maxdepth=4) }}
+                <!-- Menu area end -->
+            </div>
+            
+            <!-- Main contents area -->
+            <div class="col-sm-8 col-sm-offset-1 page white-bg">
+                {%- if pagename != "search" %}
+                <form class="search" action="{{ pathto('search') }}" method="get" _lpchecked="1">
+                    <input type="text" name="q" placeholder="Type here to search docs..." id="search">
+                    <input type="hidden" name="check_keywords" value="yes">
+                    <input type="hidden" name="area" value="default">
+                </form>
+                {%- endif %}
+                <!-- DOCS PAGE DATA HERE -->
```

Comment:
please standardize on a single start/end comment style and make all of these comments follow it